src/prepack-node.js

```diff
@@ -17,12 +17,11 @@ import { InitializationError } from "./prepack-standalone";
 import { prepackNodeCLI, prepackNodeCLISync } from "./prepack-node-environment";
 
 import type { Options } from "./options";
-import { defaultOptions } from "./options";
 
 export * from "./prepack-standalone";
 export * from "./prepack-node-environment";
 
-export function prepackStdin(options: Options = defaultOptions, callback: Function) {
+export function prepackStdin(options: Options, callback: Function) {
```

Comment:
I don't think Options should ever be required in the public API. The default should do something useful so that it is easy to try something out - even if that is just to set up the pipeline from one output to another. As you're getting something set up there are usually many other things to think about so you just want to try out those things before thinking about which configuration to use.

Ideally we would auto-detect the best approach based on the environment used.

If the new mode is safer to get started with, then that can be the default but there should at least be some default.